src/components/atoms/ProgressBar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import cx from 'classnames';
+import React from 'react';
+
+export type ProgressBarProps = {
+  totalsteps?: string;
+  stepsready?: string;
+  label?: string;
+};
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  totalsteps,
+  stepsready,
+  label,
+}) => {
+  const StepsReadyPercente = (Number(stepsready) / Number(totalsteps)) * 100;
+
+  return (
+    <div>
+      <span className="p-2.5 text-sm font-semibold text-slate-50">
+        {label} {stepsready}/{totalsteps}
+      </span>
```

Comment:
Add a new prop showLabel to do something like:

```suggestion
      {showLabel && (
        <span className="p-2.5 text-sm font-semibold text-slate-50">
          {label} {stepsready}/{totalsteps}
        </span>
       )
```